util/reggen/reg_top.sv.tpl

```diff
@@ -718,11 +735,11 @@ ${finst_gen(sr, field, finst_name, fsig_name, fidx)}
   assign racl_error_log_o.racl_role  = racl_role;
 
   if (EnableRacl) begin : gen_racl_log
-    assign racl_error_log_o.ctn_uid        = top_racl_pkg::tlul_extract_ctn_uid_bits(tl_i.a_user.rsvd);
-    assign racl_error_log_o.read_not_write = tl_i.a_opcode == tlul_pkg::Get;
+    assign racl_error_log_o.ctn_uid     = top_racl_pkg::tlul_extract_ctn_uid_bits(tl_i.a_user.rsvd);
+    assign racl_error_log_o.access_type = tl_i.a_opcode == tlul_pkg::Get;
```

Comment:
Sorry, this name is still not clear enough. If access_type is boolean means for example access_type == false for a write ??

If you name this "read_access" then a bool would make sense, true for reads, false for writes. Ideally this would be an enum... up to you.

Reply:
Fair enough. I changed to `read_access`. I left it as a `logic` atm since the only assignment to it is via `assign racl_error_log_o.read_access = tl_i.a_opcode == tlul_pkg::Get;`.